### ISSUE TITLE
🚀 A cross-entropy loss that can balance multi-task learning

### ISSUE THREAD

**Issue by raynardj** [raynardj/forgebox#9]
# 🚀 A cross-entropy loss that can balance multi-task learning

The original cross entropy loss will be adjusted with a lambda weight. The adjusted loss will be around the same scale for different tasks with different number of classes. 

https://github.com/raynardj/forgebox/blob/cec9194a52cd3061a5b1797dbddb3b9832d641d2/forgebox/multitask_ce.py#L23-L25
